test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Image.html

```diff
@@ -9,7 +9,7 @@
 
     <h2>Image Tag Helper Test</h2>
     <!-- Plain image tag -->
-    <img src="/images/red.png" alt="Red block" title="&lt;the title>">
+    <img src="/images/red.png" alt="Red block" title="&lt;the title>" />
```

Comment:
This looks like part of the test. Hence the comment `Plain image tag`. It's not modified by the normal `ImageTagHelper`